express/properties/non_scalar/inchi_key.py

```diff
@@ -0,0 +1,14 @@
+from express.properties.non_scalar import NonScalarProperty
```

Comment:
We should consider putting these into a separate folder like in ESSE: https://github.com/Exabyte-io/esse/tree/e525cd12bfd0efb038e9364c48e2d1a210ee4014/schema/properties_directory

Reply:
something along the lines of `structural`?